src/Stryker.Core/Stryker.Core/Mutants/MutationContext.cs

```diff
@@ -0,0 +1,49 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="MutationContext.cs" company="Stryker-Mutator.Net">
```

Comment:
Is this necessary? I personally would prefer not having this clutter in source files. Copyright does not need to be defined explicitly to be valid, you of course hold copyright on any material you write and we don't try to take this from you in any way. It's all publicly visible in the github logs. I will accept it if you do feel it's necessary of course since it is your work.

Reply:
Is it necessary? Actually it is, if you want the source code to be compliant with Apache 2 license (any license actually). Otherwise, the file is assumed to be unlicensed. 

My IDE automatically add this when I create a new file and I only made very basic cleanup. 

I don’t really care for my name being there or not. No prolific you want me to take the banner out, but I urge you to consider having one in your source files.

Reply:
Mhm I was not aware this is recommended so I looked it up. The way I read it, it doesn't mean the file is unlicensed but it is recommended to add it explicitly so the license information is more likely to be included when a single file is shared. The shared file would still fall under apache2 but the sharer would not necessarily know they are non-compliant with our license. Is that correct? In any case I'm not falling over whichever name is in the text I just believe it's unnecessary clutter. I am not a legal expert though so now idk if we should have it or not..

Reply:
I understand  this part as the recommendation to enclose some information within each file. But this is a should, not a must.
>The text should be enclosed in the appropriate
>     comment syntax for the file format. We also recommend that a
>    file or class name and description of purpose be included on the
 >   same "printed page" as the copyright notice for easier
 >  identification within third-party archives.

I found this [thread](https://softwareengineering.stackexchange.com/questions/125836/do-you-have-to-include-a-license-notice-with-every-source-file) on this very topic.
I just pushed a commit without the header. 
I still think you should have some minimal header info, such as project 'StrykerMutator.Net', license Apache 2 and maybe a year

Reply:
I think it could be a good idea. I think I'd like this to be automated maybe. I'll add it to the backlog. Thanks!